cypress/e2e/sign-in.cy.ts

```diff
@@ -24,7 +24,7 @@ describe('signin spec', () => {
     cy.signIn('test', '12345678');
 
     // Verify Homepage after signin
-    cy.get('h1').should('contain', 'Welcome John Doe');
+    cy.get('h1').should('contain', 'My Dashboard');
```

Comment:
We should have these Cypress tests run for each PR so we can catch these.

Reply:
Yeah I would like to do that, the only issue is we need to have the app deployed somewhere to run against. Thought about deploying to GitHub pages, but the root URL changes a bit and would need some fine-tuning.